lib/ex_machina/ecto.ex

```diff
@@ -66,6 +70,33 @@ defmodule ExMachina.Ecto do
     |> drop_ecto_fields
   end
 
+  @doc """
+  Same as `params_for/2`, but it inserts all associations and sets the
+  foreign_keys.
```

Comment:
I think that it should only insert the `belongs_to` associations and not `has_many`. What do you think?

`Same as `params_for/2`, but it inserts all belongs_to associations and sets the foreign keys`